server/scsynth/SC_Lib_Cintf.h

```diff
@@ -43,7 +44,7 @@ extern Malloc gMalloc;
 extern HashTable<class SC_LibCmd, Malloc> *gCmdLib;
 
 void initialize_library(const char *mUGensPluginPath);
-void deinitialize_library();
+SCSYNTH_DLLEXPORT_C void deinitialize_library();
```

Comment:
a private header, defining public API? this is bad. imho, `World_Cleanup` should take care of everything related to cleaning up libscsynth